src/main/java/mods/railcraft/common/util/misc/MiscTools.java

```diff
@@ -50,10 +50,6 @@ public static void registerTrack(EnumTrack rail) {
             }
     }
 
-    public static String cleanTag(String tag) {
-        return tag.replaceAll("[Rr]ailcraft\\p{Punct}", "").replaceFirst("^tile\\.", "").replaceFirst("^item\\.", "");
```

Comment:
Ok...now I'm confused. I'm pretty sure that removing this will break pretty much everything.